.github/README.md

```diff
@@ -35,11 +35,15 @@ Our user guide is at [tabbycat.readthedocs.io](http://tabbycat.readthedocs.io/).
 
 Tabbycat can be deployed in a number of ways. While you can set it up to [run on your own computer](https://tabbycat.readthedocs.io/en/stable/install/local.html) most users will want to run it as a website.
 
-Tabbycat is configured for a '1 click' install to the [Heroku](https://www.heroku.com) web platform. Click this button to begin:
+[Calico](https://calicotab.com/) is Tabbycat's managed hosting service, offering websites for a flat fee and handling all the technical aspects for you. Click this button to deploy to Calico:
+
+[![Deploy](https://raw.githubusercontent.com/gist/tienne-B/fc04ecd3c11a38424b642b4bba60e8d9/raw/b2c71d7d6a0d368d3e9dfd8002af729d155ad09b/calicodeploy.svg)](https://calicotab.com/tournaments/new/)
```

Comment:
I've had a go at making a more Heroku-like button (attached below). Would you prefer to use that? 

[button.svg.zip](https://github.com/TabbycatDebate/tabbycat/files/7242662/button.svg.zip)

Reply:
I'm partial to it. The logo is too similar (like a copy-cat) and the font becomes avec serif if Inter isn't installed (or with a different name)

Reply:
Ah, I can change the text to render as an outline rather than with the font. Do you have any ideas for a logo substitute? Maybe just the arrow without the hexagon?

Reply:
That'd be good.

Reply:
Ok, updated. If the new version works for you feel free to merge the PR as a whole!

[button.svg.zip](https://github.com/TabbycatDebate/tabbycat/files/7270893/button.svg.zip)